docs/user/run-shiny-server-operation.md

```diff
@@ -0,0 +1,62 @@
+# How To Run the Riboviz Interactive Data Visualization On Your Data
+
+This page describes how to generate and run an interactive shiny app to visualize and explore your data after running the riboviz workflow. Riboviz 2.0 has an integrated, but optional data visualization tool written in `R` and using `shiny`. The script is located at `riboviz/rscripts/run_shiny_server.R` and takes a single mandatory argument - a YAML file, and one optional argument - a port number for port forwarding. The YAML file is not the one used for a riboviz run, but instead is one called `interactive_viz_config.yaml` which will be located in your output folder.
```

Comment:
Remove mention of "2.0". This feature now applies to all releases from the current one.

Reply:
Rename `riboviz/rscripts/run_shiny_server.R` to `rscripts/run_shiny_server.R`. Keep references to file paths within the scope of the RiboViz repository except where necessary (e.g. if referring to absolute paths).